### DISCUSSION TITLE
Potentially off-topic: Is it okay to ask user question about specific set of extensions here ?

### DISCUSSION THREAD

**Discussion by bdutta** [microsoft/vscode-discussions#358] [Extension Development QnA] [answered]
This might be off-topic, because it seems that this forum is a Q&A only for people who develop extensions, and not who use extensions. SO seems to be meant more for users, but topics around vscode don't seem to be getting enough love lately, as I find quite a few questions with zero answers (but could be the Christmas/New Year season slowdown). 

Thus curious if this, or another forum more active where users could get answers about behavior of specific "interacting" (possibly) group of extensions ? For instance the Microsoft C/C++ extension set, and Code-Running. I do have a [question posted on the topic on SO already](https://stackoverflow.com/questions/74969073/should-debugger-be-launched-always-in-vscode-when-only-running-a-file-not-debug).

**Comment by burkeholland** [comment, marked as answer]
Based on your SO post, I think that this is best addressed as an issue on the Code Runner repo. What do you think?
